robot_skills/test/test_collect_arm_requirements.py

```diff
@@ -35,7 +36,8 @@ def __init__(self, robot):
             **self.REQUIRED_ARM_PROPERTIES), "None of the available arms meets all this class's" \
                                              "requirements: {}".format(self.REQUIRED_ARM_PROPERTIES)
 
-    def execute(self, userdata):
+    @staticmethod
+    def execute():
```

Comment:
Dit mag niet, je moet wel `userdata` meegeven volgens.

Reply:
Wat bedoel je met "mag niet"? De test succeed namelijk wel, maar als het bad practice is dan zal ik userdata toevoegen.